content/22.vaccines.md

```diff
@@ -304,22 +304,67 @@ Limited distribution was shared with the UAE [@url:https://www.nytimes.com/inter
+While there have been other potential protein subunit vaccines for SARS-CoV investigated _in vivo_ [@10.3389/fimmu.2020.602256; @doi:10.3389/fmicb.2020.00298], none of these candidates have successfully completed clinical trials, more than likely due to the fact that the SARS-CoV epidemic mostly ended by May 2004, thus there was less of a demand for SARS-CoV vaccine research. 
+
+Similar vaccine candidates have emerged that target the RBD found in the S1 subunit of the trimeric MERS-CoV S protein, which binds to dipeptidyl-peptidase 4 (DPP4 also known as hCD26), the entry point through which MERS-CoV infects cells [@doi:https://doi.org/10.1038/nature12005; @doi:10.1038/nature12328; @doi:10.1128/JVI.01277-13].
+After initially determining that an RBD subunit candidate (S377-588-Fc) could elicit neutralizing antibodies [@doi:10.1371/journal.pone.0081587], a study in mice determined that the administration of three sequential doses of RBD-Fc vaccine coupled with MF59, a squalene immunogenic adjuvant, induced humoral and systemic immunity in mice [@doi:10.1038/cmi.2015.03].
+Ten days post vaccination, mice that had been transduced with Ad5-hCD26 and were subsequently challenged with MERS-CoV 5 days after did not show evidence of viral infection in the lungs versus control mice [@doi:10.1038/cmi.2015.03].
+Other variations of this vaccine approach include a stable S trimer vaccine whereby proline-substituted variants of S2 can maintain a stable prefusion conformation of the S2 domain [@10.3389/fimmu.2020.602256].
+This approach leads to broad and potent neutralizing antibodies [@10.3389/fimmu.2020.602256]. 
+
+Other strategies investigating the potential use of the full length S DNA have also been investigated in mice and rhesus macaques, which elicited immune responses [@doi:10.1038/ncomms8712], but these responses were not as effective as the combination of S DNA and the S1 subunit protein combined [@doi:10.1038/ncomms8712; @doi:10.1016/j.antiviral.2016.07.015].
+Similarly to the SARS-CoV vaccine candidates, the MERS-CoV protein subunit vaccine candidates generally target the RBD [@doi:10.3389/fmicb.2020.00298; @doi:10.1016/j.vaccine.2006.10.031; @doi:10.1016/j.ebiom.2015.08.031; @doi:10.1371/journal.pone.0112602; @doi:10.1016/j.vaccine.2018.02.065; @doi:10.1016/j.virol.2016.10.005], with some targeting the full length S protein [@doi:10.1073/pnas.1707304114], and non RBD protein fragments such as the SP3 peptide [@doi:10.1089/vim.2014.0080] and the recombinant N-terminal domain (rNTD) [@doi:10.1016/j.vaccine.2016.11.064].
+No protein subunit vaccine for MERS-CoV has progressed beyond preclinical research to date. 
+
+More than 20 protein subunit vaccines have entered clinical trials for COVID-19 since the beginning of the pandemic by companies such as Sanofi and GlaxoSmithKline, Nanogen, and the Serum Institute of India [@url:https://covid19.trackvaccines.org/vaccines/].
```

Comment:
I actually don't know because I have mostly seen it as GSK, but I didn't abbreviate this time as it is first mention in the paper. I imagine they follow the same convention.